sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -105,6 +105,18 @@ class BlobClient(StorageAccountHostsMixin):  # pylint: disable=too-many-public-m
         account URL already has a SAS token. The value can be a SAS token string, and account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
         If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword int max_block_size: The maximum upload size used for a block blob. If the
+        block blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
+    :keyword int max_single_put_size: If the blob size is less than max_single_put_size, then the blob will be
+        uploaded with only one http PUT request. If the blob size is larger than max_single_put_size,
+        the blob will be uploaded in chunks. Defaults to 64*1024*1024.
+    :keyword int min_large_block_upload_threshold: The minimum upload size
+        threshold used for a block blob. Defaults to 4*1024*1024+1.
+    :keyword bool use_byte_buffer: Use a byte buffer for block blob uploads. Defaults to False.
+    :keyword int max_page_size: The maximum upload size used for a page blob. If the size of
+        the page blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
+    :keyword int max_single_get_size: The maximum size used for downloading a single blob. Defaults to 32*1024*1024.
```

Comment:
max_page_size: The maximum chunk size for uploading page blob. Defaults to 4 * 1024 * 1024.
max_single_get_size: the maximum size for a blob to be downloaded in a single call, the exceeded part will be downloaded in chunks(could be parallel)